jobs/JGLOBAL_ARCHIVE_VRFY

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 
-source "${HOMEgfs}/ush/jjob_header.sh" -e "arch_vrfy" -c "base arch_vrfy wave"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "arch_vrfy" -c "base arch_vrfy"
```

Comment:
Was "wave" here erroneously?

Reply:
Probably the same reason it was added is the same as above, although it's a little less obvious for the cause.

Reply:
Can we source `config.wave` and the ush script inside of an if-block? Is that an acceptable solution for the g-w core dev team?

Reply:
I ran a test C48_S2SW case on C6 and verified that the `VRFY` job does not need to source `config.wave`.